lib/fetch/index.js

```diff
@@ -1044,7 +1042,7 @@ function fetchFinale (fetchParams, response) {
       //    and responseStatus.
       if (fetchParams.request.initiatorType != null) {
         // TODO: update markresourcetiming
-        markResourceTiming(timingInfo, fetchParams.request.url, fetchParams.request.initiatorType, globalThis, cacheState, bodyInfo, responseStatus)
+        markResourceTiming(timingInfo, fetchParams.request.url.href, fetchParams.request.initiatorType, globalThis, cacheState, bodyInfo, responseStatus)
```

Comment:
Can we detect if somebody is listening to these resourceTiminig? This seems like an expensive call that we could potentially skip if there is no one watching.